debian/changelog

```diff
@@ -1,3 +1,41 @@
+ubuntu-advantage-tools (27.15) mantic; urgency=medium
+
+  * d/ubuntu-advantage-tools.postinst:
+    - more specific regex for ua_config warning
+  * New upstream release 27.15 (LP: #2017949)
+    - apport: collect default log files if present for bug reports
+    - apt messaging: add bionic-specific urls
+    - config: no longer load uaclient.conf from current working directory
+    - fix:
+      + add support for --no-related flag
+      + separate target USN from related USNs
+    - general:
+      + logs to user cache directory when run as non-root
+      + fix bug where non-root commands failed with file permission error
+        accessing /tmp/ubuntu-advantage (GH: #2567)
+      + use system environment vars by default in sub processes
```

Comment:
I think we missing the github issue for this entry:
https://github.com/canonical/ubuntu-pro-client/issues/2527

Reply:
Thanks! Just fixed